src/Http/CookieFactory.php

```diff
@@ -85,6 +94,15 @@ public function make($name, $value = null, $maxAge = null)
             $cookie = $cookie->withDomain($this->domain);
         }
 
+        // Explicitly set SameSite value, use sensible default if no value provided
+        if ($this->samesite === 'strict') {
+            $cookie = $cookie->withSameSite(SameSite::strict());
+        } elseif ($this->samesite === 'none') {
+            $cookie = $cookie->withSameSite(SameSite::none());
+        } else {
+            $cookie = $cookie->withSameSite(SameSite::lax());
+        }
```

Comment:
How about this? If an incorrect value is given the code will immediately error out as well.

```suggestion
        $cookie = $cookie->withSameSite(forward_static_call([SameSite, $this->samesite ?? 'lax']));
```

Also setting a default in line 67 then isn't required, just keep it null so that we know it isn't set at all because you already define a fallback here.

Reply:
I'm happy with this, wasn't even ware that forward_static_call was a thing, great feature to know about for things like this though.